### PR TITLE
#4 Success message

### PR DESCRIPTION
**Pull request by alihammad99**
I have added additional styles as style.css file.
I have added a success message for the form and confirmation.

### CODE REVIEW

File: starterOnly/index.html [hunk 1]
```diff
@@ -57,6 +57,9 @@ <h1 class="hero-headline">
         <div class="content">
           <span class="close" onclick="closeWindow()"></span>
           <div class="modal-body">
+            <span id="form-success-message"> Thank you for<br>
```
Comment (pratapdd) [inline starterOnly/index.html:60]: use lesser <br /> tags

File: starterOnly/style.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    color: red;
+    font-weight: thin;
+  }
+  .checkbox2-label {
+    margin-top: 11px;
+  }
+  div.content {
+    min-height: 625px;
+  }
+  #close-btn {
```
Comment (pratapdd) [inline starterOnly/style.css:13]: try to use classname and not ids'

File: starterOnly/index.html (hunk 1 above)
Comment (alihammad99) [reply starterOnly/index.html:60]: <br> tags just removed

File: starterOnly/style.css (hunk 2 above)
Comment (alihammad99) [reply starterOnly/style.css:13]: Changing to classes make errors at the moment

**Pull request merged**